website/src/examples/mapbox.js

```diff
@@ -43,7 +43,7 @@ class MapboxDemo extends Component {
   }
 
   componentDidUpdate(prevProps) {
-    if (this.props.data && !prevProps.data) {
+    if (this.props.data && !prevProps.data && this._view) {
```

Comment:
It looks like `renderToDOM` never returns anything ...

https://github.com/visgl/deck.gl/blame/5aef80773cd47d58eccc3e35481bd765f26fcb5f/examples/website/mapbox/app.jsx#L143-L150

... so `this._view` is never defined? I don't know if it should block this PR, but something seems broken here.

Related:

- https://github.com/visgl/deck.gl/pull/8558

Reply:
I thought `renderToDOM` is only used when running the example standalone (see index.html in the example's folder), whereas the website imports `App` directly.